test/parallel/test-whatwg-url-parsing.js

```diff
@@ -26,7 +26,7 @@ const failureTests = tests.filter((test) => test.failure).concat([
 ]);
 
 const expectedError = common.expectsError(
-  { code: 'ERR_INVALID_URL', type: TypeError }, 110);
+  { code: 'ERR_INVALID_URL', type: TypeError }, 113);
```

Comment:
I think the counter here is just `failureTests.length`, there is no need to hard-code this

Reply:
agree, done.